tests/test_indexer.py

```diff
@@ -330,6 +294,70 @@ def get_search_results(cls, query, expected_hit_count):
             else:
                 time.sleep(0.5)
 
+    def create_sample_bundle_created_event(self, bundle_key):
+        raise NotImplemented()
+
+    def process_new_indexable_object(self, event, logger):
+        raise NotImplemented()
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
I prefer having the setup/teardown methods near the top of the class ... that's just where I am used to looking for them. This change is optional, at your discretion.

Reply:
moved setup and teardown to top